profile/README.md

```diff
@@ -1,4 +1,7 @@
 # OpenSTEF - Open Short Term Energy Forecasting
+[![License: MPL2.0](https://img.shields.io/badge/License-MPL2.0-informational.svg)](https://github.com/alliander-opensource/power-grid-model/blob/main/LICENSE)
```

Comment:
Is it possible we get this license from openstef. Now it refers to power-grid-model?